documentation/API_documentation/NumberVerificationdevice_phone_number_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# Number Verification Device Phone Number API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to retrieve the phone number associated with the line from which the call was made, so that I can ensure that I obtain the correct phone number and avoid identity theft fraud. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br>-Returns the phone number associated with the access token so API clients can get the number and verify it themselves. |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:BusinessManager has successfully subscribed to the Number Verification product from the product catalog.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer:user performs an authorization request to CSP</li><li>The means to get the access token are known to the Customer:User to ensure secure access of the API.|

```

Comment:
The means to get the access token are known to the Customer:User to ensure secure access of the API.
--> Access token generation initiated by Customer:User is based on network authentication to ensure secure access of the API